src/experiments/story-ad-auto-advance.js

```diff
@@ -25,29 +35,22 @@ export const StoryAdAutoAdvance = {
   TEN_SECONDS: '31060907',
 };
 
-/** @const Object<string, string> */
-export const AdvanceExpToTime = {
-  '31060905': '6s',
-  '31060906': '8s',
-  '31060907': '10s',
+/** @const @private {!ExperimentInfoDef} */
```

Comment:
is there a reason to hoist this? it isn't reused anywhere, I believe colocated with its usage was a better place to put it. 

nit: the name should also have `experiment` in it somewhere

Reply:
Moved back into function

Reply:
thank you for doing this at both locations